pydantic/main.py

```diff
@@ -106,7 +101,10 @@ def __new__(
         Returns:
             type: the new class created by the metaclass
         """
-        if _base_class_defined:
+        # Note `ModelMetaclass` refers to `BaseModel`, but is also used to *create* `BaseModel`, so we rely on the fact
+        # that `BaseModel` itself won't have any bases, but any subclass of it will, to determine that the `__new__`
```

Comment:
```suggestion
        # that `BaseModel` itself won't have any bases, but any subclass of it will, to determine whether the `__new__`
```
I think this reads a bit better since the next line checks that it's _not_ the BaseModel class, as phrased it reads to me like it's confirming we are doing BaseModel, which confused me for a sec.